Mathlib/Algebra/BigOperators/Finsupp.lean

```diff
@@ -249,12 +249,12 @@ theorem sum_apply [Zero M] [AddCommMonoid N] {f : α →₀ M} {g : α → M →
   finset_sum_apply _ _ _
 
 -- Porting note: inserted ⇑ on the rhs
-theorem coe_finset_sum [AddCommMonoid N] (S : Finset ι) (f : ι → α →₀ N) :
+@[simp, norm_cast] theorem coe_finset_sum [AddCommMonoid N] (S : Finset ι) (f : ι → α →₀ N) :
```

Comment:
Given that the only downstream change from this is to require _removing_ the simp attribute, this change isn't clearly positive. Can you justify why the regression is acceptable?

Reply:
Actually, there is no regression anymore. The proof was just suboptimally written, with `← IH` going against the confluence